src/views/News/components/NewsList/index.jsx

```diff
@@ -117,21 +133,13 @@ NewsList.propTypes = {
       // Expiration: PropTypes.string.isRequired,
     }),
   ).isRequired,
-
-  personalUnapprovedNews: PropTypes.arrayOf(
-    PropTypes.shape({
-      SNID: PropTypes.number.isRequired,
-      Subject: PropTypes.string.isRequired,
-      ADUN: PropTypes.string.isRequired,
-      Entered: PropTypes.string.isRequired,
-      categoryName: PropTypes.string.isRequired,
-      Body: PropTypes.string.isRequired,
-      // Expiration: PropTypes.string.isRequired,
-    }),
-  ).isRequired,
-
+  header: PropTypes.string.isRequired,
   handleNewsItemEdit: PropTypes.func.isRequired,
   handleNewsItemDelete: PropTypes.func.isRequired,
+  handleNewsApprovalStatus: PropTypes.func.isRequired,
+  unapproved: PropTypes.any,
+  isAdmin: PropTypes.bool.isRequired,
+  defaultExpanded: PropTypes.bool,
```

Comment:
Also this could be `isExpandedByDefault`. I think I'm understanding what this field does, but correct me if I'm wrong.

Reply:
When I tried the Tab component, `isExpandedByDefault` is no longer needed.